### ISSUE TITLE
Introduce an operation to add fields to records.

### ISSUE THREAD

**Issue by vjovanov** [scala-records/scala-records#97]

**Comment by cvogt** [comment]
Is there anything more than union of two records needed to add fields to one?

**Comment by vjovanov** [comment]
Not really. The biggest problem here is how to make it configurable for different back-ends. The front-end is not complicated.

**Comment by cvogt** [comment]
What do you mean by configurable for different backends?

**Comment by vjovanov** [comment]
Well for Spark we directly map values to existing records. For regular records we use a HashMap.
